### PR TITLE
4-week: wrong

### PR DESCRIPTION
**Pull request by olavea**
Is this right formatting?

### CODE REVIEW

File: content/olaVeaMail/2022/2022-01-25-wrong/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: This Sunday’s Skill-Builder-Session went WRONG!
+emojii: ⛵️
```
Comment (raae) [inline content/olaVeaMail/2022/2022-01-25-wrong/index.md:3]: No need for this line, the code adds it as its the same every time.

File: content/olaVeaMail/2022/2022-01-25-wrong/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+Ship Ahoy Skill-Builder!
```
Comment (raae) [inline content/olaVeaMail/2022/2022-01-25-wrong/index.md:6]: No need for this line, the code adds it.

**Review by raae — CHANGES_REQUESTED** [review]
Move the content so both image and index.md can be found in `content/olavea-emails/2022-01-25-wrong/`.

Check to see if everything works, it should show up at `http://localhost:8000/emails/olavea/`

File: content/olaVeaMail/2022/2022-01-25-wrong/index.md (hunk 2 above)
Comment (olavea) [reply content/olaVeaMail/2022/2022-01-25-wrong/index.md:6]: Ship Ahoy Skill-Builder?

Comment (raae) [reply content/olaVeaMail/2022/2022-01-25-wrong/index.md]: Both Skip Ahoy Skill-Builder and the emoji

**Pull request merged**